config/common/errors.go

```diff
@@ -93,6 +93,11 @@ var (
 
 	// Kernel arguments
 	ErrGeneralKernelArgumentSupport = errors.New("kernel argument customization is not supported in this spec version")
+
+	// Selinux Module
+	ErrContentInvalid = errors.New("Content is empty, please provide content.")
+	ErrNameInvalid    = errors.New("Name is empty, please provide a valid name.")
+	ErrFieldInvalid   = errors.New("Please, provide valid information.")
```

Comment:
If these are all specific to SELinux, we should probably prefix them with e.g. `ErrSelinux`.

`ErrFieldInvalid` seems redundant.

For the error messages, let's use the same wording as other errors, e.g. "field ... is required".

Reply:
Nice! I will change it! Thanks.